### DISCUSSION TITLE
WiFi Spy Microphone

### DISCUSSION THREAD

**Discussion by FernandoJose2** [BruceDevices/firmware#1280] [General]
What are your comments about this feature? I think, it will be very cool in James Bond 007 movies.

We no need a external module for this. Cardputer already has a esp32, sdcard module and mic.

WiFi Spy Microphone With ESP32

https://www.instructables.com/Spy-Microphone-Connected/

https://www.hackster.io/mvtdesign/wifi-spy-microphone-with-esp32-devkit-a02120

https://www.hackster.io/mvtdesign/esp-wifi-spy-microphone-esp32-and-esp8266-80d3a5

https://wicard.net/esp-spy-microphone/

**Comment by LEGOATDU49** [comment]
Yes, like this we could hide the device in a place and hear the person live via the webui or record the audio on the SD card

**Comment by Millyvanilly1988** [comment]
There is already one:
https://github.com/geo-tp/M5Cardputer-Audio-Stream-Server
But it would be nice to have it in Bruce firmware

**Comment by uncooldude** [comment]
Agreed, We really need this feature, u can make different versions of bruce, like burce-pentesting core (which will have the pentesting tools), and other distro with smthing crazy like this

**Comment by eadmaster** [comment]
this library supports various audio codecs and streaming:
- https://github.com/pschatzmann/arduino-audio-tools/
- https://github.com/pschatzmann/arduino-audio-tools/wiki/Communication
- https://github.com/pschatzmann/arduino-audio-tools/blob/main/examples/examples-communication/http-server/streams-i2s-webserver_wav/streams-i2s-webserver_wav.ino

it could replace ESP8266Audio, since it provides the same functionality.

**Comment by archiepa** [comment]
Will it be added though?

**Reply by eadmaster** [reply]
looks doable in 16KHz wav format only without any extra library.

**Comment by b-host** [comment]
it would be great to have this feature!

**Comment by Huzzla101** [comment]
Man....That's like a moist vag.... 📞

On Thu, Sep 4, 2025 at 12:50 AM blinhares ***@***.***> wrote:

> it would be great to have this feature!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/pr3y/Bruce/discussions/1280#discussioncomment-14301119>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BLDL6BRBAZUDIELJSMBA3Y33Q5WCVAVCNFSM6AAAAAB6HW43DGVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTIMZQGEYTCOI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Kind Regards

C.R

May Prosperity allways be with us!!

**Comment by Nickviscy** [comment]
+1 
We need this
